app/forms/base_work_form.rb

```diff
@@ -13,6 +13,9 @@ class BaseWorkForm < Reform::Form
 
   property :work_type, on: :work_version
   property :version_description, on: :work_version
+  property :new_user_version, virtual: true # in the future, will need to be persisted to the work_version for drafts
```

Comment:
I'm not totally clear on this. Can you explain more?

Reply:
In the situation where the user makes a selection whether this will be a new version or not and then does a "Save as draft" there needs to be a way to retain that selection for when they return to the draft. This ticket doesn't implement that--it's just noting that we have more work to do to retain this selection. Goal was to keep the ticket as small as possible since user versions aren't fully implemented and public yet.